RecoHI/Configuration/python/customise_PPwithHI.py

```diff
@@ -137,18 +139,52 @@ def storeCaloTowersAOD(process):
 
     return process
 
+# Add Centrality reconstruction in pp reco
+def customiseRecoCentrality(process):
+
+    process.load('RecoHI.HiCentralityAlgos.pACentrality_cfi')
+    process.pACentrality.producePixelTracks = cms.bool(False)
+
+    process.recoCentrality = cms.Path(process.pACentrality)
+
+    process.schedule.append(process.recoCentrality)
+
+    return process
+
+
+# Add ZDC, RPD and Centrality to AOD event content
+def storePPbAdditionalAOD(process):
+
+    process.load('Configuration.EventContent.EventContent_cff')
+
+    # extend AOD content
+    if hasattr(process,'AODoutput'):
+        process.AODoutput.outputCommands.extend(['keep *_zdcreco_*_*'])
+        process.AODoutput.outputCommands.extend(['keep ZDCDataFramesSorted_hcalDigis_*_*'])
+        process.AODoutput.outputCommands.extend(['keep ZDCDataFramesSorted_castorDigis_*_*'])
+        process.AODoutput.outputCommands.extend(['keep recoCentrality*_pACentrality_*_*'])
+
+    if hasattr(process,'AODSIMoutput'):
+        process.AODSIMoutput.outputCommands.extend(['keep *_zdcreco_*_*'])
+        process.AODSIMoutput.outputCommands.extend(['keep ZDCDataFramesSorted_hcalDigis_*_*'])
+        process.AODSIMoutput.outputCommands.extend(['keep ZDCDataFramesSorted_castorDigis_*_*'])
+        process.AODSIMoutput.outputCommands.extend(['keep recoCentrality*_pACentrality_*_*'])
+
+    return process
+
 def customisePPrecoforPPb(process):
- 
-     process=addHIIsolationProducer(process)
-     process=storeCaloTowersAOD(process)
- 
-     return process
- 
+
+    process=addHIIsolationProducer(process)
+    process=storeCaloTowersAOD(process)
+    process=customiseRecoCentrality(process)
```

Comment:
@mandrenguyen suggested me to merge this one and make a new PR. I just need to know whether any other changes for this PR or is it stable now?